app/models/views/applikation/result.rb

```diff
@@ -0,0 +1,28 @@
+module Views
+  module Applikation
```

Comment:
I think we don't have to use this incorrect name any more, we can use `Application` as it's nested in the `Views` namespace, so it doesn't clash with any mode.